packages/app/src/server/models/comment.js

```diff
@@ -94,6 +94,9 @@ module.exports = function(crowi) {
     return commentData;
   };
 
+  commentSchema.statics.findCommentById = async function(commentId) {
+    return this.findOne({ _id: commentId });
+  };
```

Comment:
This process is simple, so I think this method isn't needed. Please just call `findOne()` from CommentService directly.

Reply:
Removed method findCommentById and call findOne directly from CommentService